lib/config/all.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
what do we need this config for?

Reply:
Oops. 😅  That was something I experimented with, but decided there wasn't a use case for simply including all rules. I've deleted it now. Good catch!